pages/common/hub.md

```diff
@@ -11,6 +11,14 @@
 
 `hub clone {{github_username}}/{{repo_name}}`
 
+- Fork repository cloned from another user (creates a remote named after your GitHub username):
+
+`hub fork`
+
+- Create a pull request from your fork:
+
+`git push {{github_username}} && hub pull-request`
```

Comment:
How is the name of the remote decided ? Its slightly confusing when you say "username" when it actually refers to the remote. Maybe we can add some more explanation and say {{remote_name}} in the example.